CHANGELOG.md

```diff
@@ -8,97 +8,32 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [10.9.0]
 ### Uncategorized
-- Version v10.9.0
-- EIP-1559 V2: Use default gas preferences of user for all transactions ([#13238](https://github.com/MetaMask/metamask-extension/pull/13238))
 - Handle missing eth_accounts permission history in getLastConnectedInfo ([#13257](https://github.com/MetaMask/metamask-extension/pull/13257))
```

Comment:
I don't think https://github.com/MetaMask/metamask-extension/pull/13257 is end user facing